api/src/client/dynamics/housing/types.ts

```diff
@@ -22,3 +23,66 @@ export type HousingPropertyInterestInfo = (
   address: string,
   municipalityId: string
 ) => Promise<HousingPropertyInterestResponse>;
+
+export interface HotelMotelRegistrationClient {
+  getHotelMotelRegistration: (
+    accessToken: string,
+    propertyInterestId: string,
+    buildingCount: number
+  ) => Promise<HousingRegistrationRequestResponse>;
+}
+
+export type HousingRegistrationRequest = {
```

Comment:
The reason this is `HousingRegistrationRequest` and not `HousingHotelMotelRegistrationRequest` is that this can and will be re-used for the Multiple Dwelling property type in an upcoming ticket